src/components/Comments/Comments.jsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from "react";
+import { getArticleComments } from "../../utils/util-api-calls";
+import { formatDate } from "../../utils/util-data-formatters";
+import { HiArrowsUpDown, HiMiniUserCircle, HiCalendarDays, HiChevronDown, HiChevronUp, HiChatBubbleLeftEllipsis } from "react-icons/hi2";
+import "./Comments.css";
+
+function Comments({ article_id }) {
+	const [comments, setComments] = useState([]);
+	const [commentsLoading, setCommentsLoading] = useState(true);
+  const [commentsVisible, setCommentsVisible] = useState(true);
+
+	useEffect(() => {
+		setCommentsLoading(true);
+		getArticleComments(article_id)
+			.then((res) => {
+				setCommentsLoading(false);
+				setComments(res.data.comments);
+			})
+			.catch((error) => {
+				console.log(error);
+			});
+	}, []);
+
+  useEffect(()=>{setCommentsVisible(false)}, [])
+
+	return commentsLoading ? (
+		<h2>Loading ...</h2>
```

Comment:
Excellent to see some loading feedback specifically for the comments let us know something is happening 👍